tailwind.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
           "message-arrow":
             "var(--color-message-arrow, var(--color-border-frame, var(--color-border-base, #000)))", // message arrow head
           "message-hover":
-            "var(--color-text-message-hover, var(--color-bg-base, #fff))",
+            "rgb(var(--color-text-message-hover, var(--color-bg-base, 255 255 255)))",
```

Comment:
We should keep it consistent between this file and tailwind.css on `rgb(255 255 255)` vs `rgb(255,255,255)`. In tailwind.css, variables are defined as `255,255,255`.